files/en-us/web/api/range/clonecontents/index.html

```diff
@@ -14,16 +14,17 @@
   domxref("DocumentFragment") }} copying the objects of type {{ domxref("Node") }}
   included in the {{ domxref("Range") }}.</p>
 
-<p>Event Listeners added using DOM Events are not copied during cloning. HTML attribute
-  events are duplicated as they are for the DOM Core cloneNode method. HTML id attributes
+<p>Event listeners added using {{domxref("EventTarget.addEventListener()", "addEventListener()")}}
+  are not copied during cloning. HTML attribute events are duplicated as they are
+  for the {{ domxref("Node.cloneNode()") }} method. HTML <code>id</code> attributes
   are also cloned, which can lead to an invalid document through cloning.</p>
 
 <p>Partially selected nodes include the parent tags necessary to make the document
   fragment valid.</p>
 
 <h2 id="Syntax">Syntax</h2>
 
-<pre class="brush: js"><em>documentFragment </em>= <em>range</em>.cloneContents();
+<pre class="brush: js"><var>documentFragment</var> = <var>range</var>.cloneContents();
```

Comment:
```suggestion
<pre class="brush: js">documentFragment = range.cloneContents();
```
Any markup inside `pre` elements won’t survive HTML➡Markdown conversion, so we might as well go ahead and just remove it.

Reply:
thanks!